src/lib/components/PublishButton/SubmitReviewButton.js

```diff
@@ -68,6 +70,13 @@ class SubmitReviewButtonComponent extends Component {
 
     const { isConfirmModalOpen } = this.state;
 
+    const btnLblSubmitReview = isRecordSubmittedForReview
+      ? i18next.t("Submitted for review")
+      : i18next.t("Submit for review");
+    const btnLbl = directPublish
```

Comment:
```suggestion
    const buttontnLabel = directPublish
```
to improve the readability